form/form_collections.rst

```diff
@@ -340,6 +340,31 @@ you'll replace with a unique, incrementing number (e.g. ``task[tags][3][name]``)
         // Add the new form at the end of the list
         $collectionHolder.append($newFormLi)
     }
+    
+Code without jQuery library
+
+.. code-block:: javascript
+    const newItem = (e) => {
```

Comment:
Maybe renaming `newItem()` as `addNewItem()` would look better?

Reply:
The method is renamed